### DISCUSSION TITLE
Ubuntu 22.04: Error adding apt repo

### DISCUSSION THREAD

**Discussion by meb427** [open-meteo/open-meteo#1030] [General]
May be I should open an issue but there is an issue getting Ubuntu 22.04 keys

sudo gpg --keyserver hkps://keys.openpgp.org --no-default-keyring --keyring /usr/share/keyrings/openmeteo-archive-keyring.gpg  --recv-keys E6D9BD390F8226AE
echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/openmeteo-archive-keyring.gpg] https://apt.open-meteo.com $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/openmeteo-api.list

sudo apt update
sudo apt install openmeteo-api

Seem there is a typo already in hkps://keys.openpgp.org

**Comment by patrick-zippenfenig** [comment]
Seems to work for me. `hkps` should be correct for the key exchange protocol. Do you get an error?

```bash
# gpg --keyserver hkps://keys.openpgp.org --no-default-keyring --keyring /usr/share/keyrings/openmeteo-archive-keyring.gpg --recv-keys E6D9BD390F8226AE
gpg: keybox '/usr/share/keyrings/openmeteo-archive-keyring.gpg' created
gpg: key E6D9BD390F8226AE: public key "Open-Meteo <info@open-meteo.com>" imported
gpg: Total number processed: 1
gpg:               imported: 1
```

**Comment by meb427** [comment]
Didn't work for my on 2 Ubuntu 22.04 and the apt update was getting error like the key invalid. 
I will give it another try on new VM's. But I had to remove repo/key then add again the repo with the exact key. 
Was very odd error.
